core/new-gui/src/app/dashboard/user/component/user-avatar/user-avatar.component.ts

```diff
@@ -53,11 +25,7 @@ export class UserAvatarComponent implements OnInit {
     if (userName.length <= 5) {
       return userName;
     } else {
-      return this.getUserInitial(userName).slice(0, 5);
+      return userName.slice(0, 5);
```

Comment:
can we keep the logic for username abbr?

Reply:
The original `getUserInitial` function was unnecessary and wrong.